exporter/elasticsearchexporter/logs_exporter.go

```diff
@@ -109,6 +112,14 @@ func (e *elasticsearchLogsExporter) pushLogRecord(ctx context.Context, resource
 		fIndex = fmt.Sprintf("%s%s%s", prefix, fIndex, suffix)
 	}
 
+	if e.logstashFormat.Enabled {
```

Comment:
according to this logic, I thinks the three config `logs_index`, `logs_dynamic_index`, `logstash_format` have a priority order among them, right? The `logstash_format` will overwrite other two config if it is enabled?

Reply:
yes, as I described in docs: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/90c73e8e7a3d575c365fe9b1da49aa1db70697e6/exporter/elasticsearchexporter/README.md?plain=1#L48

Reply:
Initially, I designed `logstash_format` to be incompatible with `trace/logs_index` and `logs_dynamic_index` in order to reduce complexity.  But we will need to write the following index: `otlp-logs-{service-name}-2023.12.03`. So, I changed the logical here.

Reply:
Ok, but in unit test `TestGetSuffixTime`, it seems that there is not case for all `logs_index`, `logs_dynamic_index` `logstash_format ` are enable, the index name will be like `otlp-logs-{service-name}-2023.12.03`. 

If I'm right, could you add one?

Reply:
okay

Reply:
Others are lgtm.